src/main/java/com/moviecat/servicesImpl/genre/GenreServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
I am not sure exactly why we need a `GenreServiceImpl` class.
Kindly explain.

Reply:
When we setup code climate to monitor the efficiency of our codebase, it is clear that by the time we finish with all the entities, the class `RPCRouter` will be too congested and will have too much lines of code which will be denied quality by code climate. So I implemented this outside of the `RPCRouter` class so that I can just call it in few lines inside the `RPCRouter` class.